Justfile

```diff
@@ -37,6 +37,14 @@ lint-links:
     		--exclude twitter.com --exclude explorer.optimism.io --exclude linux-mips.org --exclude vitalik.eth.limo \
     		--exclude-mail /input/README.md "/input/specs/**/*.md"
 
+# Validates spelling using cspell
+lint-specs-spelling:
+    npx cspell "./**/*.md"
+
+# Updates cspell words file with new words
+lint-specs-spelling-fix:
+    npx cspell --words-only --unique "./**/*.md" | sort --ignore-case | uniq > words.txt
```

Comment:
Should we lowercase the entries so we can avoid all the casing variations in words.txt?

Reply:
cspell is case sensitive which probably makes sense for some proper nouns. I think it's likely best to leave it as-is.